witwidget/notebook/jupyter/js/package.json

```diff
@@ -22,13 +22,11 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
-    "file-loader": "^3.0.1",
     "rimraf": "^2.6.1",
     "webpack": "^3.5.5"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1.10 || ^2",
-    "@jupyter-widgets/jupyterlab-manager": ">=0.38.1 <2.0.0"
```

Comment:
Could you list which platforms you tested this on, just to make sure this doesn't have any side effect?

Reply:
tested in latest jupyter plain version, and locally installed jupyterlab 2.2.9 (latest), and cloud AI platform notebook (which is jupyterlab 1.2).